src/public/CowSwap.json

```
@@ -1397,6 +1397,14 @@
       "chainId": 100,
       "logoURI": "https://raw.githubusercontent.com/cowprotocol/token-lists/main/src/public/images/100/0xf490c80aae5f2616d3e3bda2483e30c4cb21d1a0/logo.png"
     },
+    {
+      "address": "0xfecb3f7c54e2caae9dc6ac9060a822d47e053760",
+      "symbol": "BRLA",
+      "name": "BRLA Digital BRLA",
```

Update with name coming from contract

```suggestion
      "name": "BRLA Token",
```